src/popover/types.js

```diff
@@ -219,8 +219,8 @@ export type AnchorPropsT = {
   onBlur?: (e: Event) => mixed,
   onClick?: (e: Event) => mixed,
   onFocus?: (e: Event) => mixed,
-  onMouseEnter?: (e: Event) => mixed,
-  onMouseLeave?: (e: Event) => mixed,
```

Comment:
Please restore these prop types. I would like to maintain the current API. I think even after it has changed to pointer events the top level api describes essentially the same behavior

Reply:
Consider if an application creates a custom Anchor component rather than providing an element https://codesandbox.io/s/base-web-statefulpopover-forked-hs1t44?file=/src/example.js

Reply:
@chasestarr restored the prop types. Please review the changes. Thanks! 😄